e2e/wdio/core/tests/flexible-column-layout.e2e-spec.ts

```diff
@@ -182,4 +192,18 @@ describe('Flexible column layout component test', () => {
         click(exitExampleBtn);
         expect(doesItExist(column)).toBe(false);
     }
+
+    function checkExistSeparator(section: string): void {
+        click(section + button);
+        click(columnButton2);
+        pause(1000);
```

Comment:
why pause instead of doing a wait ?
expect(waitForDisplayed(separator)).toBe(true)
or expect(waitForPresent(....   if issue with selector existing